### PR TITLE
Fix links in create-sharepoint-home-site-for-viva-connections.md

### PR DESCRIPTION
**Pull request by dariomws**
Closes https://github.com/MicrosoftDocs/Viva/issues/348.

### CODE REVIEW

File: Viva/connections/create-sharepoint-home-site-for-viva-connections.md [hunk 1]
```diff
@@ -45,23 +45,23 @@ A home site needs site navigation that is organized well, highlights popular res
...
 
 ## Build the site 
-Start with a [modern SharePoint communication site](https://support.microsoft.com/en-us/office/create-a-communication-site-in-sharepoint-7fb44b20-a72f-4d2c-9173-fc8f59ba50eb#:~:text=Steps%20to%20create%20a%20communication%20site%3A%201%20Sign,5%20Give%20your%20new%20communication%20site%20a%20name). Next, [start adding sections](https://support.microsoft.com/en-us/office/add-sections-and-columns-on-a-sharepoint-modern-page-fc491eb4-f733-4825-8fe2-e1ed80bd0899)following the diagram in the site sections summary. Then, add and edit web parts. The guidance for customizing web parts can be applied when you are in edit-mode for each web part. 
+Start with a [modern SharePoint communication site](https://support.microsoft.com/office/create-a-communication-site-in-sharepoint-7fb44b20-a72f-4d2c-9173-fc8f59ba50eb). Next, [start adding sections](https://support.microsoft.com/office/add-sections-and-columns-on-a-sharepoint-modern-page-fc491eb4-f733-4825-8fe2-e1ed80bd0899) following the diagram in the site sections summary. Then, add and edit web parts. The guidance for customizing web parts can be applied when you are in edit-mode for each web part.
```
Comment (JohanFreelancer9) [inline Viva/connections/create-sharepoint-home-site-for-viva-connections.md:53]:
```suggestion
Start with a [modern SharePoint communication site](https://support.microsoft.com/office/create-a-communication-site-in-sharepoint-7fb44b20-a72f-4d2c-9173-fc8f59ba50eb). Next, [start adding sections](https://support.microsoft.com/office/add-sections-and-columns-on-a-sharepoint-modern-page-fc491eb4-f733-4825-8fe2-e1ed80bd0899) following the diagram in the site sections summary. Then, add and edit web parts. The guidance for customizing web parts can be applied when you are in edit mode for each web part. 
```

File: Viva/connections/create-sharepoint-home-site-for-viva-connections.md [hunk 2]
```diff
@@ -85,7 +85,7 @@ Start with a [modern SharePoint communication site](https://support.microsoft.co
 
 ## Customize site details
 
-After the site is built, edit site details to create a customized design that aligns with your organizations brand and identity. These [site details can be managed](https://support.microsoft.com/en-us/office/customize-your-sharepoint-site-320b43e5-b047-4fda-8381-f61e8ac7f59b#:~:text=1%20Customize%20the%20theme.%20You%20can%20change%20the,links%20on%20your%20site%20navigation.%20For%20more%20) from **Settings** > **Change the look**.
+After the site is built, edit site details to create a customized design that aligns with your organizations brand and identity. These [site details can be managed](https://support.microsoft.com/office/customize-your-sharepoint-site-320b43e5-b047-4fda-8381-f61e8ac7f59b) from **Settings** > **Change the look**.
```
Comment (JohanFreelancer9) [inline Viva/connections/create-sharepoint-home-site-for-viva-connections.md:88]:
```suggestion
After the site is built, edit site details to create a customized design that aligns with your organization's brand and identity. These [site details can be managed](https://support.microsoft.com/office/customize-your-sharepoint-site-320b43e5-b047-4fda-8381-f61e8ac7f59b) from **Settings** > **Change the look**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - two minor additional edits, please.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@Holland-ODSP PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Review by Holland-ODSP — APPROVED** [review]

**Pull request merged**